packages/components/src/tools-panel/tools-panel-header/component.js

```diff
@@ -21,6 +21,7 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 		menuLabel,
 		resetAll,
 		toggleItem,
+		hasSelectedMenuItems,
```

Comment:
Tiniest of tiny nits: the other identifiers in this list (aside from the one below) are in alphabetical order. Shall we move this one up to just beneath `hasMenuItems`?